src/components/search/Search.jsx

```diff
@@ -154,11 +154,20 @@ const Search = () => {
   const shouldDisplayBalanceAlert = hasNoEnterpriseOffersBalance || hasLowEnterpriseOffersBalance;
 
   const { content_type: contentType } = refinements;
-  const hasRefinements = Object.keys(refinements).filter(refinement => refinement !== 'showAll').length > 0 && (contentType !== undefined ? contentType.length > 0 : true);
+  const hasRefinements = Object.keys(refinements).filter(refinement => refinement !== 'showAll').length > 0
+    && (contentType !== undefined ? contentType.length > 0 : true);
 
-  const optimizelyPrequerySuggestionClickHandler = (courseKey) => {
-    if (isExperimentVariation) {
-      pushEvent(EVENTS.PREQUERY_SUGGESTION_CLICK, { courseKey });
+  const isPreQueryEnabled = enterpriseConfig.enterpriseFeatures?.featurePrequerySearchSuggestions
+    && isExperimentVariation;
+
+  const optimizelySuggestionClickHandler = (courseKey) => {
+    // Programs pass in a list of keys. Optimizely does not accept array values
+    // so we are joining the items in the array.
+    const courseKeyMetadata = Array.isArray(courseKey) ? courseKey.join(', ') : courseKey;
+    if (isPreQueryEnabled) {
```

Comment:
Make sure we're covering both cases in unit tests
![image](https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/322346/08801557-2cd2-40ce-b653-fa5ed070eeb3)